spacy_wordnet/wordnet_domains.py

```diff
@@ -68,16 +79,43 @@ def wordnet_synsets_for_domain(self, domains: List[str]):
         ]
 
     @staticmethod
-    def __find_synsets(token: Token, lang: str):
+    def __find_synsets(token: Token,
+                       lang: str,
+                       pos: Optional[Union[str, List[str]]] = None) -> List[Synset]:
+        if pos is None:
+            pos = ["verb", "noun", "adj"]
+
+        elif isinstance(pos, str):
+            pos = [pos]
+
+        elif not isinstance(pos, list):
+            try:
+                pos = list(pos)
+
+            except TypeError:
+                raise TypeError("pos argument must be None, type str, or type"
+                                "list.")
+
+        acceptable_pos = {"verb": VERB, "noun": NOUN, "adj": ADJ}
+
+        # check if any element in `pos` is not in `acceptable_pos`
+        if set(pos).difference(acceptable_pos):
+            raise ValueError("pos argument must be a combination of 'verb', "
+                             "'noun', or 'adj'.")
+
+        token_pos: List[int] = [acceptable_pos[k] for k in pos]
```

Comment:
```suggestion
    def __find_synsets(token: Token,
                       lang: str,
                       pos: Optional[Union[str, List[str]]] = None) -> List[Synset]:
        if pos is None:
            pos = []
        elif isinstance(pos, str):
            pos = [pos]
        elif not isinstance(pos, list):
            try:
                pos = list(pos)
            except TypeError:
                raise TypeError("pos argument must be None, type str, or type list.")
        
        acceptable_pos = {"verb": VERB, "noun": NOUN, "adj": ADJ} # We can define this as a private class constant
        # check if any element in `pos` is not in `acceptable_pos`
        if set(pos).difference(acceptable_pos):
            raise ValueError("pos argument must be a combination of 'verb', "
                             "'noun', or 'adj'.")

        token_pos: List[int] = [acceptable_pos[k] for k in pos]
        if not token_pos:
           token_pos = [token.pos]
```

Reply:
Okay I understand now! I can get behind this suggestion. I'm going to update the docstring and unit test.